tacs/elements.pyx

```diff
@@ -789,9 +846,11 @@ cdef class Quad16NonlinearShell(Element):
     .. seealso:: :ref:`theory/shell_element:Mixed Interpolation of Tensorial Components`
 
     .. note::
-        varsPerNode: 6
+        **varsPerNode**: 6
 
-        outputElement: ``TACS.BEAM_OR_SHELL_ELEMENT``
+        **numNodes**: 16
```

Comment:
Are these meant to show up in the docs? I couldn't see them when I tried building the docs locally

Reply:
They should show up here: https://smdogroup.github.io/tacs/core/elements.html#tacs.elements.Quad4Shell

Reply:
I'm not seeing the `numNodes` note you're adding here when I build the docs locally (I'm assuming just checking out this branch without recompiling is all I need to do right?)

![image](https://github.com/smdogroup/tacs/assets/20371123/4ab7bf7e-2b09-447a-ba82-c5551306e3f5)

Reply:
No, you need to recompile cython (make interface) and the docs to update